packages/components/modules/messages/hooks/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { ChatRoomHeaderFragment$data } from '../../../__generated__/ChatRoomHeaderFragment.graphql'
+
+export const isGroupChat = (chatRoom: ChatRoomHeaderFragment$data) => chatRoom.title !== null
```

Comment:
This should be updated when a `isGroupChat` field is added on the backend.

Reply:
can we add a TODO so we don't lose track of it